Lib/csv.py

```diff
@@ -129,18 +126,13 @@ def __next__(self):
                 d[key] = self.restval
         return d
 
-    __class_getitem__ = classmethod(types.GenericAlias)
```

Comment:
Does the removal of `__class_getitem__` help to fix the issue? If it's unrelated, we need a separate GitHub issue and another pull request for it.

Reply:
Ok. We juste have one question, because, we don't know how to test all of this without pull requesting every time. Is there a way to do it ?